specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/examples/Patches_Apply.json

```diff
@@ -0,0 +1,88 @@
+{
+  "operationId": "Patches_Apply",
+  "title": "Patches_Apply_0",
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder",
+    "patchName": "testPatch-25fe4b",
+    "api-version": "2023-08-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/builders/testBuilder/patches/testPatch-25fe4b",
+        "name": "testPatch-25fe4b",
+        "type": "Microsoft.App/builders/patches",
+        "properties": {
+          "targetEnvironmentId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/test-env",
+          "targetContainerAppId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/containerApps/test-app",
+          "targetRevisionId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/apps/test-app/revisions/test-app--jm3vvry",
+          "provisioningState": "Succeeded",
+          "patchApplyStatus": "Succeeded",
+          "createdTime": "2022-10-10T12:06:20.3421+00:00",
+          "updatedTime": "2022-10-10T12:06:20.3421+00:00",
+          "patchDetails": [
```

Comment:
>patchDetails [](http://example.com/codeflow?start=11&length=12)

looking at the example not sure why all these needed to be in a separate details bag

Reply:
sorry i've just realized this one cannot be flattened since patchDetails array length is more than 1 when dealing with multiple containers in a revision. Are there other ways to flatten it in your opinion?

Reply:
Likely not, thanks for pointing out its actually a collection.